src/quo2/components/calendar/calendar/month_picker/utils.cljs

```diff
@@ -0,0 +1,38 @@
+(ns quo2.components.calendar.calendar.month-picker.utils
+  (:require [utils.i18n :as i18n]))
+
+(def ^:const months-with-year
+  "Maps the corresponding string representation of a month
+   By it's numeric index as in cljs-time"
+  {1  "january-year"
+   2  "february-year"
+   3  "march-year"
+   4  "april-year"
+   5  "may-year"
+   6  "june-year"
+   7  "july-year"
+   8  "august-year"
+   9  "september-year"
+   10 "october-year"
+   11 "november-year"
+   12 "december-year"})
+
+(defn format-month-year
+  [year month]
+  (let [month (cond
+                (or (nil? month) (zero? month)) 1
+                (> month 12)                    12
+                :else                           month)]
+    (str (i18n/label (get months-with-year month) {:year year}))))
```

Comment:
I don't understand the need for `months-with-year` @mohsen-ghafouri, nor the translations named `january-year`, `february-year`, and so on.

We can just do something like the code below. Seems better than having to rely on  manual translations.

```clojure
;; Added to utils.datetime ns
(def format-long-month
  (memoize (fn [month]
             (.format ^js ((get-formatter-fn (constantly "MMMM")))
                      (t/date-time 1970 month)))))

(format-as-long-month 3) ; => "March"

;; ------------

;; quo2.components.calendar.calendar.month-picker.utils
(defn format-month-year
  [year month]
  (let [month (cond
                (or (nil? month) (zero? month)) 1
                (> month 12)                    12
                :else                           month)]
    (str (utils.datetime/format-long-month month) " " year)))
```

Reply:
thanks, applied and removed month translations